pkgs/tools/bluetooth/blueman/default.nix

```diff
@@ -31,10 +31,11 @@ in stdenv.mkDerivation rec {
 
   configureFlags = [ (lib.enableFeature withPulseAudio "pulseaudio") ];
 
-  preFixup = ''
+  # This mimics ../../../development/interpreters/python/wrap.sh
+  postFixup = ''
     makeWrapperArgs="--prefix PATH ':' ${binPath}"
```

Comment:
Just my two cents here, but wrapsPythonProgramsIn is the right place to add $out to the pythonPath. The interface is error prone otherwise. May I propose 
```diff
diff --git a/pkgs/development/interpreters/python/wrap.sh b/pkgs/development/interpreters/python/wrap.sh
index 1c74e612b5..f86738f4a0 100644
--- a/pkgs/development/interpreters/python/wrap.sh
+++ b/pkgs/development/interpreters/python/wrap.sh
@@ -1,7 +1,7 @@
 # Wrapper around wrapPythonProgramsIn, below. The $pythonPath
 # variable is passed in from the buildPythonPackage function.
 wrapPythonPrograms() {
-    wrapPythonProgramsIn "$out/bin" "$out $pythonPath"
+    wrapPythonProgramsIn "$out/bin"
 }
 
 # Builds environment variables like PYTHONPATH and PATH walking through closure
@@ -41,7 +41,7 @@ patchPythonScript() {
 # suffix).
 wrapPythonProgramsIn() {
     local dir="$1"
-    local pythonPath="$2"
+    local pythonPath="${2:-$out $pythonPath}"
     local f
 
     buildPythonPath "$pythonPath"
diff --git a/pkgs/tools/bluetooth/blueman/default.nix b/pkgs/tools/bluetooth/blueman/default.nix
index 8512cd4fcb..60686764ec 100644
--- a/pkgs/tools/bluetooth/blueman/default.nix
+++ b/pkgs/tools/bluetooth/blueman/default.nix
@@ -34,8 +34,8 @@ in stdenv.mkDerivation rec {
   postFixup = ''
     makeWrapperArgs="--prefix PATH ':' ${binPath}"
     # This mimics ../../../development/interpreters/python/wrap.sh
-    wrapPythonProgramsIn "$out/bin" "$out $pythonPath"
-    wrapPythonProgramsIn "$out/libexec" "$out $pythonPath"
+    wrapPythonProgramsIn "$out/bin"
+    wrapPythonProgramsIn "$out/libexec"
   '';
 
   meta = with lib; {
```
or maybe `local pythonPath="$out ${2:-$pythonPath}"` ?